projects/openshift.py

```diff
@@ -18,37 +18,92 @@
 # Refer to the README and COPYING files for full details of the license
 #
 
+import sys
+
 from fabric.api import cd
-from fabric.api import put
+from fabric.api import env
 from fabric.api import run
 from fabric.api import settings
 from fabric.api import sudo
 from fabric.api import task
-from fabric.api import env
-from fabric.contrib.files import sed
 
-from common import install_packages
 from common import enable_services
 from common import enable_tcp_ports
+from common import install_packages
+from manageiq import create_provider
+from manageiq import refresh_provider
 
 
+DEFAULT_ATTEMPTS = 10
+PROVIDER_NAME = 'openshift01'
+
 @task(name='install-deps')
 def install_deps():
     install_packages([
-        'docker-io',
+        'docker',
+        'docker-registry',
         'git',
         'golang',
         'screen',
         'vim',
     ])
+    # required on CentOS7
+    sudo('yum -y update device-mapper')
 
 
 @task
-def deploy(proxy=''):
+def deploy():
     install_deps()
 
-    enable_services(['docker.socket'])
+    enable_services(['docker.service'])
+
     enable_tcp_ports([8443, 8444])
 
-    sudo('usermod -a -G docker {0}'.format(env.user))
+    sudo('groupadd docker')
+    sudo('usermod -aG docker {0}'.format(env.user))
+
     run('test -d origin || git clone https://github.com/openshift/origin.git')
+
+
+@task
+def start():
+    with cd('origin'):
+        run('make')
+        run('./_output/local/go/bin/openshift start --write-config=true '
+            '--node-config=true '
+            '--master-config=openshift.local.certificates/master/master'
+            '.yaml')
```

Comment:
Keep the filename on the same line (`openshift.local.certificates/master/master.yaml`)